x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/DriverContext.java

```diff
@@ -33,11 +35,30 @@
  */
 public class DriverContext {
 
+    /** A default driver context. The returned bigArrays is non recycling. */
+    public static DriverContext DEFAULT = new DriverContext(BigArrays.NON_RECYCLING_INSTANCE);
```

Comment:
It's not really a "default" - it's more of a "don't use this in production" kind of thing.

Reply:
Or, like, I guess a "temporary" kind of thing.